### ISSUE TITLE
NavigationInfo @type is a JSON String not a JSON array.  Data from browser.toJSONString()

### ISSUE THREAD

**Issue by coderextreme** [create3000/x_ite#146]
![image](https://github.com/create3000/x_ite/assets/705783/61dfaf33-d28e-4422-8813-535163d3163f)

Talk to Don and Vince if you disagree.  I will back you, but it may require a few changes...

**Comment by coderextreme** [comment]
It's also possible to accept either in JSON Schema.  I will work on that too. I'll broach the discussion.

**Comment by create3000** [comment]
The issue is not whether NavigationInfo.type should be coded differently, but whether MFString values should be coded differently, and that will probably not be the case.

**Comment by coderextreme** [comment]
I agree, but no one wants to discuss it frankly or honestly, they just avoid, avoid, avoid. So I'm looking to supporting both in JSON schema, just for NavigationInfo @type.  I agree that all MFStrings should be coded the same.

**Comment by coderextreme** [comment]
JSON schema has been tested, but I haven't changed the schema generation script yet.

**Comment by coderextreme** [comment]
If you change MFString implementation, let me know.  I am currently assuming that MFStrings are arrays.  I have 2 branches of code, one, .json, with NavigationInfo.type as a string.  The other, .x3dj as an array.  I can merely substitute array for string in the schema if and when the time is right.

**Comment by coderextreme** [comment]
Current X3dtoJson.xslt documentation has NavigatorInfo@type as an array.

**Comment by coderextreme** [comment]
I've got changes to X3dToJson.xslt if you are interested.  I will be pushing this through my pipelines.

**Comment by coderextreme** [comment]
I’ll work on maintaining tools such that that is possible, including
supporting stylesheets and even X3DJSAIL.  I’m going to check Aaron’s work,
hopefully soon.

John

**Comment by coderextreme** [comment]
I have not run any checks.  I need to download X3DJSAIL and the various
stylesheets.

I will reopen the ticket if need be.

John
